docs/user-documentation/transcripts.md

```diff
@@ -0,0 +1,46 @@
+# Transcripts
+
+Transcripts, captions, or subtitles may be added to audio and video media.
+This functionality is available because of a field type, "Media track", 
+provided by the Islandora module, and the "Audio with Captions" and "Video 
+with Captions" field formatters from the Islandora Audio and Video modules, 
+respectively. 
+
+To add transcripts (or subtitles, or captions) to a Repository Item 
+using Islandora Defaults, navigate to the Service File media (or whichever one
+will be playing), open the edit form, and in the "Track" field, add a WebVTT file 
```

Comment:
In order to se the "Track" field for uploading the .vtt file, I first had to alter the Structure > Media types > Edit Audio > Manage form display options to move the Track field from the disabled section to enable it. This was in the sandbox. Not sure if it makes sense to add this note before the instructions to edit the service file?

Reply:
Yep - addressed in latest commit.